### ISSUE TITLE
unable to get token from the generated link

### ISSUE THREAD

**Issue by valotee** [prasmussen/gdrive#656]
Google Auth used has been deprecated; see the link below since Oct 3

https://developers.google.com/identity/protocols/oauth2/resources/oob-migration

**Comment by jknechtel** [comment]
Yeah, same issue here. Not sure whether @prasmussen will have some bandwidth for this in near future or whether someone else from the community should take this up?
Thanks!

**Comment by jknechtel** [comment]
FYI, I've managed to get it working via another fork. See https://github.com/prasmussen/gdrive/issues/645

**Comment by Msameim181** [comment]
You can use this [Repository](https://github.com/Msameim181/gdrive-cli-builder). Don't forget to hit Star.

**Comment by hfcheng66** [comment]
> You can use this [Repository](https://github.com/Msameim181/gdrive-cli-builder). Don't forget to hit Star.
Thank you! I tried to build a windows version, however still getting the same error. It seems the config/credential is saved somewhere that needs to be deleted? Do you know how to fix this?
